src/main/java/emissary/output/DropOffUtil.java

```diff
@@ -995,6 +999,82 @@ public Date parseEventDate(final String dateString, final boolean supplyDefaultO
         return date;
     }
 
+    public static class ArrayListMap<K, V> extends AbstractMap<K, V> {
```

Comment:
Why are we creating this map class? Also this doesn't seem to be a map. This has a lookup speed of O(n), whereas maps are expected to have a lookup of O(1)